proposals/XXXX-incremental-heartbeat-assisted-presence.md

```diff
@@ -19,57 +19,53 @@
 
-High level overview goes into a higher level overview of this MSC and goes into background. This exists
-for the dual purposes of helping make the MSC easier to write as it allowed a outline to be created
-and it also hopes to help make the proposal accessible. As the high level overview explains the system without
-getting too concerned with the minute details found in the proposal section.
-
-### Background 
-
-[Presence](https://spec.matrix.org/v1.11/client-server-api/#presence) is only enabled by homeserver admins who have the resources to burn on it. 
-All homeserver implementations recognise presence as a very inefficient feature, and also a feature that does not scale. 
-In addition, the current presence system is fragile: there have been presence bugs in sliding sync proxy, and presence flooding 
+[Presence](https://spec.matrix.org/v1.11/client-server-api/#presence) is only enabled by homeserver admins who have the resources to burn on it.
+All homeserver implementations recognise presence as a very inefficient feature, and also a feature that does not scale.
+In addition, the current presence system is fragile: there have been presence bugs in sliding sync proxy, and presence flooding
 can be accidentally triggered by clients.
 
 ### High level overview of how to fix this.
 
+The High level overview goes into a higher level overview of this MSC and goes into background.
+The overview is provided for the purpose of making the MSC easier to write, and to make the proposal easier to understand.
+As the high level overview explains the system without getting too concerned with the minute details found in the proposal section.
+
 This MSC proposes fixing presence by taking a page out of the routing protocol world. The precursor to
 this MSC in the form of the gist from earlier exists due to my studies having us learn the ins and outs of
 the EIGRP and OSPF routing protocols. It turns out that these routing protocols have a similar problem to us.
 They need to distribute Alive or Dead status to all the nodes so your routing tables are up to date.
 
-And what is presence if not Alive or Dead status information. Now OSPF takes the problematic but works at limited
-enough scale approach of sending a complete copy of state every time anything changes. Its a known flaw but it works. 
+And what is presence if not Alive or Dead status information? Now OSPF takes the problematic-but-works-at-limited
+enough-scale approach of sending a complete copy of state every time anything changes. Its a known flaw but it works.
 Now EIGRP uses incremental updates and that's exactly what i propose we copy. This MSC therefore proposes
-that we change presence to be a incremental system that only communicates state changes and move expiry into a 
-separate subsystem of presence.
+that we change presence to be a incremental system that only communicates state changes and move expiry into a
+separate subsystem of presence, we have called the heartbeat system.
```

Comment:
I am not completely sure im happy with this but my inclanation is F it. Looks good enough to me.